src/lib/components/paginator.svelte

```
@@ -26,14 +27,15 @@
             | undefined;
         offset?: number;
         children: Snippet<[T[], number]>;
+        [key: string]: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)